src/Database/Connection.php

```diff
@@ -735,6 +736,9 @@ public function transactional(callable $transaction)
 
         try {
             $result = $transaction($this);
+        } catch (Throwable $e) {
```

Comment:
If we catch `Throwable` then don't need to catch `Exception` below, right?

Reply:
We do, since `Throwable` doesn't exist in PHP 5.6 (this PR is for 3.x).

Reply:
We do. Since this version needs PHP5 support.
See the link/example I posted in the ticket.
There are like 5-10 other examples btw that might also need a look in terms of this issue, and if it could be relevant to also apply this exception handling there.

Reply:
Got it. I missed it was for 3.x.